{{cookiecutter.repo_name}}/CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!--
+The rules for this file:
+  * entries are sorted newest-first.
+  * summarize sets of changes - don't reproduce every git log comment here.
+  * don't ever delete anything.
+  * keep the format consistent (79 char width, M/D/Y date format) and do not
```

Comment:
If I could turn back time, I would have used [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) for our dates instead of the weirdest format possible, M/D/Y ;-)

Reply:
Oops, changed!